linode/manager
# test: [M3-8661] - Cypress test to validate CAA record are editable

## Description 📝

Added Cypress test to validate CAA record are editable

## Changes  🔄

Added below cypress tests under in spec file - `cypress/e2e/core/domains/smoke-create-domain-records.spec.ts`

<div class="table-wrap">

TC_ID | TC_NAME | Steps | Expected Result
-- | -- | -- | --
TC_1 | Verify CAA record are editable with **iodef** tag | 1. Prerequites -  Prerequites -  use createDomain() to create domain.<br>2. Navigate to Domain details Landing page. | 1. Button "Add a CAA Record" is present. <br>2. Validate creation of the CAA record with **iodef** tag. <br>3. Validate that it should update valid record and not allow invalid record. <br> 4. Validate that it should throw a error "You have entered an invalid target" when update with invalid record.
</div>

- checks CAA record should update valid record and not allow invalid record 

## How to test 🧪

`yarn cy:run -s cypress/e2e/core/domains/smoke-create-domain-records.spec.ts`

### Verification steps

When test executed using this command `yarn cy:run -s cypress/e2e/core/domains/smoke-create-domain-records.spec.ts` all tests should pass successfully.

![Tests](https://github.com/user-attachments/assets/c63528c0-660b-49f4-a356-215c66eca56b)

## As an Author I have considered 🤔

*Check all that apply*
